backend/src/main/java/com/redhat/cloud/notifications/NotificationsApp.java

```diff
@@ -22,11 +26,24 @@
 
     private static final Logger LOG = Logger.getLogger(NotificationsApp.class);
 
+    @Inject
+    MeterRegistry registry;
+
+    AtomicInteger started;
+
+    @PostConstruct
+    void postConstruct() {
+        started = registry.gauge("notifications.backend.started", new AtomicInteger(0));
```

Comment:
Why do we need a gauge instead of a counter?

Reply:
A counter can onle be incremented but not decremented.

Reply:
I know what a gauge is, but I don't see why it's needed here.

Reply:
I mean its value will be `1` for a few milliseconds at most and `0` during something like 99% of the startup time.

Reply:
exactly. I want it to use to create the blue line here: https://www.inovex.de/wp-content/uploads/2020/10/example_simple.png